src/pruner_util.cpp

```diff
@@ -1,14 +1,29 @@
 #include <fstream>
+#include <boost/random.hpp>
 
 #include "frontends/p4/toP4/toP4.h"
 
 #include "pruner_util.h"
 
 namespace P4PRUNER {
 
-int get_exit_code(cstring name, P4PRUNER::PrunerOptions options) {
+static boost::random::mt19937 rng;
```

Comment:
Native rand() is not great in CPP so we are using the Mersenne Twister from Boost. With that we can also use a reliable seed.